### ISSUE TITLE
Add build information to README

### ISSUE THREAD

**Issue by pdefreitas** [jernejsk/OpenELEC-OPi2#43]
I think that it would be useful, Orange PI forum is super slow from my location and getting info from github is way easier.

**Comment by pdefreitas** [comment]
The same thing for prebuilt images, they should be marked as releases and shared here on github :)

**Comment by jernejsk** [comment]
Why should prebuild images marked as releases? They are not good enough by my standards...

If you can't access orange pi forum use some free web proxy like hide.me. It also helps to use different ISP (for example, I can always access opi forum through my mobile data plan while not always through my home ISP).

I really don't see a point to have same informations on multiple places and forum seems like much better place to put ton of informations.

**Comment by pdefreitas** [comment]
I have to disagree since people browser the source code here on Git. It's a pain to have to open the webbrowser just to check building instructions for a specific device.

**Comment by jernejsk** [comment]
But then, why didn't official OpenELEC team put instructions inside? All informations can be found on their wiki...
